cli/src/main/java/com/liferay/blade/cli/BladeCLI.java

```diff
@@ -401,6 +408,52 @@ private static String _extractBasePath(String[] args) {
 		return defaultBasePath;
 	}
 
+	private static String _extractProfileName(String[] args) {
+		String defaultProfile = null;
+
+		if (_isInitCommand(args) && (args.length > 2)) {
```

Comment:
Lets move all of this logic into the `InitCommand`.  I want to keep BladeCLI agnostic as to what commands are executing.  BladeCLI shouldn't "know" about InitCommand.